DotNet Examples/Test Monitor/Delete Results/Delete Results.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Rename the project file and folder into **DeleteResults**

Reply:
updated the project name